app/services/idv/flows/doc_auth_flow.rb

```diff
@@ -26,8 +26,8 @@ class DocAuthFlow < Flow::BaseFlow
         { name: :getting_started },
         { name: :verify_id },
         { name: :verify_info },
-        { name: :secure_account },
         { name: :get_a_letter },
+        { name: :secure_account },
```

Comment:
Would this reordering need to be conditional based on `IdentityConfig.store.gpo_personal_key_after_otp` ?